src/ldes/LDESinLDP.ts

```diff
@@ -341,4 +341,36 @@ export class LDESinLDP implements ILDES {
             }
         }
     }
+
+
+    /**
+     * Returns the last members added to the container based on the timestamp.
+     * @returns {Promise<Quad[]>}
+     */
+    public async getLastMembers(): Promise<Quad[]> {
```

Comment:
```suggestion
    public async getLastMember(): Promise<Quad[]> {
```
Should it not return the last member based on timestamp?